package.json

```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js"
+    "start": "nodemon server.js"
```

Comment:
Is nodemon included as a local dependency? If not, this will fail in Heroku.

Reply:
Nodemon should be installed globally, so we don't need to install it locally for this project. Did you have nodemon installed globally? If not then just run `node server.js` in your terminal to run localhost. The script is just a shortcut to run `npm start`.  For Heroku, I already specified `node server.js` in the Procfile when we run Heroku, so this won't affect Heroku.

Reply:
If I run `npm install`, will nodemon be installed? Because it doesn't seem to be outlined in package-lock.json. We should ensure that `npm install` includes everything that we expect to have in our local development environment.

Reply:
No, nodemon won't be installed if you run npm install, because globally installed packages won't show in (local) package.json of our project. You can go ahead and change that script back to previous one (`node server.js`). It's just a shortcut, we can also delete it.